docs/development-workflow.md

```diff
@@ -0,0 +1,52 @@
+# Development Workflow
+
+Start by forking the pd GitHub repository, make changes in a branch and then send a pull request. 
```

Comment:
The development workflow consists of the following steps:
1. Fork the pd repository.
2. Make changes in a branch.
3. Send a pull request.

Reply:
Does the `pd` need to use the uppercase? /cc @dcalvin

Reply:
I think it's OK either way here, since we use lowercase for the Repo name.

Reply:
> Does the `pd` need to use the uppercase? /cc @dcalvin

I think it's OK either way here, since we use lowercase for the Repo name.